### PR TITLE
Test OpenApi shema

### PR DESCRIPTION
**Pull request by iMalkav**
# What's new

- Add Schemathesis-based schema conformance tests for the web API. The new suite automatically covers all `GET` operations and safe `POST` operations, while destructive, session-breaking, cloud-dependent, and websocket endpoints are kept in an explicit skip list.
- Attach generated request data and response status to Allure to make schema failures easier to debug.
- Update `openapi.yaml` to better match actual runtime validation and make schema-driven testing reliable. This includes tighter validation for access keys, storage paths, brightness, timestamps, timezone names, device names, asset IDs, display text, countdown timestamps, and MAC addresses.
- Add a missing `operationId` for Wi-Fi scan, define coordinate bounds for display elements, and bump the API version to `19.0.0`.
- Strengthen frontend integration tests by adding reference refresh support for screenshot tests via `UPDATE_REFS=1`.
- Replace fragile exact back-display snapshot comparison with structural checks because the back display contains dynamic system UI.
- Expand alignment verification to the full 3x3 grid.
- Verify that image and animation `stock_path` assets actually render, instead of only checking that the request is not rejected.
- Stabilize the paused busy-timer fixture through the `ACTIVE -> PAUSED` transition and document the `FW-832` regression path with a skipped test.
- Improve test stability and observability by logging failed web requests, handling Wi-Fi `reconnecting` during cleanup, rebalancing workflow timeouts, and reducing the device reset interval to speed up retries.

# Verification

- Flash the firmware to a test device and make the web frontend available via `WEB_BASE_URL`.
- Run `python3 -m pytest tests/integration/frontend/schema_conformance/test_api_schema.py -v`.
- Run `python3 -m pytest tests/integration/frontend/display/test_api_display_draw.py -v`.
- Run `python3 -m pytest tests/integration/frontend/assets/test_display_screenshot.py -v`.
- Run `python3 -m pytest tests/integration/frontend/display/test_api_display_priority.py -v`.
- Run `python3 -m pytest tests/integration/frontend/wifi/test_api_wifi.py -v`.
- If screenshot references need to be regenerated after font or rendering changes, run `UPDATE_REFS=1 python3 -m pytest tests/integration/frontend/assets/test_display_screenshot.py -k "TestDisplayAlignment"`.
- AST syntax check passed for all modified Python test files.
- 
# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Review by ssecsd — COMMENTED** [review]
```
============================= test session starts ==============================
platform linux -- Python 3.11.15, pytest-7.4.4, pluggy-1.6.0
rootdir: /_work/bsb-test-runner-Lima/bsb-firmware/bsb-firmware/tests
configfile: pyproject.toml
plugins: hypothesis-6.151.9, xdist-3.8.0, schemathesis-3.39.16, allure-pytest-2.15.0, anyio-4.12.1, asyncio-0.23.8, subtests-0.14.2
asyncio: mode=Mode.AUTO
collected 0 items / 1 error

==================================== ERRORS ====================================
________________________ ERROR collecting test session _________________________
/opt/hostedtoolcache/Python/3.11.15/x64/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1204: in _gcd_import
    ???
<frozen importlib._bootstrap>:1176: in _find_and_load
    ???
<frozen importlib._bootstrap>:1140: in _find_and_load_unlocked
    ???
E   ModuleNotFoundError: No module named 'schemathesis.conftest'
=============================== warnings summary ===============================
conftest.py:5
  /_work/bsb-test-runner-Lima/bsb-firmware/bsb-firmware/tests/conftest.py:5: DeprecationWarning: 'telnetlib' is deprecated and slated for removal in Python 3.13
    import telnetlib  # TODO: Replace with alternative before Python 3.13 (deprecated)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
- generated xml file: /_work/bsb-test-runner-Lima/bsb-firmware/bsb-firmware/tests/test-results.xml -
=========================== short test summary info ============================
ERROR  - ModuleNotFoundError: No module named 'schemathesis.conftest'
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
========================= 1 warning, 1 error in 0.06s ==========================
```

Tests broken

File: tests/integration/frontend/schemathesis/test_api_schema.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    def test_get_conformance(self, case: schemathesis.Case) -> None:
+        """
+        Each GET endpoint must:
+        - return an HTTP status code documented in the schema
+        - return a response body that matches the schema (when a body is documented)
+        """
+        _skip_if_dangerous(case)
+
+        with allure.step(f"Call {case.method.upper()} {case.formatted_path}"):
+            response = case.call()
```
Comment (ssecsd) [inline tests/integration/frontend/schemathesis/test_api_schema.py]: Why not use existing tests.conftest.web_session fixture?

Comment (iMalkav) [reply tests/integration/frontend/schemathesis/test_api_schema.py]: add web_session in call

File: tests/integration/frontend/schema_conformance/conftest.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    validate_schema=False: the device may serve a schema with minor deviations
+    from the OpenAPI 3.1 draft — we do not want schema meta-validation errors
+    to abort the entire test run.
+    """
+    schema_url = f"{web_base_url}/openapi.yaml"
+    return schemathesis.from_uri(
+        schema_url,
+        validate_schema=False,
+    )
```
Comment (ssecsd) [inline tests/integration/frontend/schema_conformance/conftest.py]:
```suggestion
@pytest.fixture(scope="session")
def schemathesis_schema_safe_read(schemathesis_schema: schemathesis.BaseSchema):
    """Schema with dangerous operations excluded — for GET tests."""
    return schemathesis_schema.exclude(
        operation_id=lambda oid: oid in SKIP_OPERATION_IDS,
    )


@pytest.fixture(scope="session")
def schemathesis_schema_safe_write(schemathesis_schema: schemathesis.BaseSchema):
    """Schema filtered to only safe write operations."""
    return schemathesis_schema.include(
        operation_id=lambda oid: oid in SAFE_WRITE_OPERATION_IDS,
    )
```

File: tests/integration/frontend/schemathesis/test_api_schema.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    if op_id in SKIP_OPERATION_IDS:
+        pytest.skip(f"Skipped dangerous operation: {op_id}")
+
+
+def _skip_if_not_safe_write(case: schemathesis.Case) -> None:
+    """Skip POST operations not in the explicit safe-write allowlist."""
+    op_id = case.operation.definition.raw.get("operationId", "")
+    if op_id not in SAFE_WRITE_OPERATION_IDS:
+        pytest.skip(f"Not in safe-write allowlist: {op_id}")
+
```
Comment (ssecsd) [inline tests/integration/frontend/schemathesis/test_api_schema.py]: Here instead of helpers use two lazy schemas
```suggestion
_safe_read_schema = schemathesis.from_pytest_fixture("schemathesis_schema_safe_read")
_safe_write_schema = schemathesis.from_pytest_fixture("schemathesis_schema_safe_write")
```

File: tests/integration/frontend/schemathesis/test_api_schema.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    - generates boundary values for query/path parameters
+    - validates the JSON response structure against $ref schemas
+    - flags both documented and undocumented status codes
+    """
+
+    @allure.id("SCHEMA-001")
+    @allure.title("GET endpoints conform to OpenAPI schema")
+    @pytest.mark.schemathesis
+    @pytest.mark.frontend
+    @_schema.parametrize(method="GET")
```
Comment (ssecsd) [inline tests/integration/frontend/schemathesis/test_api_schema.py]:
```suggestion 
 @_safe_read_schema.parametrize(method="GET")
```

**Review by ssecsd — DISMISSED** [review]

**Comment by hedger** [comment]
Un-draft when ready

**Review by hedger — APPROVED** [review]
